<?xml version="1.0"?>
<launch>
    <node pkg="twist_controller" type="dbw_node.py" name="dbw_node">
        <param name="vehicle_mass" value="1080." />
        <param name="fuel_capacity" value="0." />
        <param name="brake_deadband" value=".2" />
        <param name="decel_limit" value="-20." />
        <param name="accel_limit" value="10." />
        <param name="wheel_radius" value="0.335" />
        <param name="wheel_base" value="3" />
        <param name="steer_ratio" value="14.8" />
        <param name="max_lat_accel" value="3." />
        <param name="max_steer_angle" value="8." />
        <param name="throttle_Kp" value="0.5" />
        <param name="throttle_Ki" value="0.00001" />
        <param name="throttle_Kd" value="0.0" />
        <param name="steering_Kp" value="0.5" />
        <param name="steering_Ki" value="0.0" />
        <param name="steering_Kd" value="0.0" />
    </node>
</launch>
